src/Mount.cpp

```diff
@@ -1812,10 +1812,10 @@ void Mount::focusSetSpeedByRate(int rate)
 // focusContinuousMove
 //
 /////////////////////////////////
-void Mount::focusContinuousMove(int direction)
+void Mount::focusContinuousMove(FocuserDirection direction)
 {
   // maxSpeed is set to what the rate dictates
-  setSpeed(FOCUS_STEPS, direction * _maxFocusRateSpeed);
+  setSpeed(FOCUS_STEPS, (int)direction * _maxFocusRateSpeed);
```

Comment:
You should use `static_cast<int>(` for compile-time type checking instead of C-style casts